frontend/src/container/NotificationWall.jsx

```diff
@@ -255,11 +256,68 @@ export class NotificationWall extends React.Component {
       }
     }
 
+    const defaultEmptyUrlMsg = props.t('This notification has no associated content')
```

Comment:
Not directly your work, but I think this method could gain to be split (it is more than 300 lines long).
Maybe split in one method by entityType?

Reply:
I thought about it several time since I've worked quite a bit on this component. I'm not sure it's a win w.r.t clarity: variables are reused across the function, and several functions require jumping through the code to read it. We can discuss if you want.